usage_examples/streamlit/streamlit_app.py

```diff
@@ -0,0 +1,49 @@
+from spacy import util
+from spacy_streamlit import visualize_ner, visualize_textcat
+import streamlit as st
+import os
+
+DEFAULT_TEXT = "Ernest Hemingway, born in Illinois, is generally considered one of the best authors of his time."
+
+st.title("spacy-llm Streamlit Demo")
+st.markdown(
+    """
+    The [spacy-llm](https://github.com/explosion/spacy-llm) package integrates 
+    Large Language Models (LLMs) into spaCy, featuring a modular system 
+    for fast prototyping and prompting, and turning unstructured responses 
+    into robust outputs for various NLP tasks, no training data required.
+
+    This demo uses the OpenAI backend to demonstrate the NER and textcat 
+    tasks.
+    """
+)
+
+os.environ["OPENAI_API_KEY"] = st.text_input("Your OpenAI API key", type="password")
```

Comment:
```suggestion
os.environ["OPENAI_API_KEY"] = st.text_input("Your OpenAI API key", type="password", value=os.environ.get("OPENAI_API_KEY", ""))
```

Reply:
Oh this makes so much more sense. Thanks!